.gitmodules

```diff
@@ -276,15 +276,15 @@
 	branch = main
 	platforms = genio
 [submodule "open-iot-sdk"]
-	path = third_party/open-iot-sdk/sdk
-	url = https://git.gitlab.arm.com/iot/open-iot-sdk/sdk.git
+	path = third_party/corstone/sdk
```

Comment:
Should we rename the submodule entry above this line ?

Reply:
Not sure, we still use the Open IoT SDK as a components provider. I guess it will be removed when the rebranding will be completed.